lib/src/multihash/varint_utils.dart

```diff
@@ -5,16 +5,28 @@ import 'package:buffer/buffer.dart';
 
 import 'models.dart';
 
+/// Maximum safe integer in JavaScript. (2^53)
+const _maxIntegerJS = 9007199254740992;
```

Comment:
For `web`, it should be (2^53) - 1, 
as per https://stackoverflow.com/a/60358200/20281592.